lib/profile/header.dart

```diff
@@ -1,63 +1,141 @@
-        color: Color(0xFF0F1236), 
+        color: kPrimaryColor, 
         borderRadius: BorderRadius.only(
           bottomLeft: Radius.elliptical(40.0, 40.0)
         ),
       ),
+      child: Container(
+        padding: EdgeInsets.only(top: 30.0),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget> [
+            buildProfilePhoto(),
+            buildFullName(),
+            buildPopUpMenuButton(context)
+          ]
+        )
+      ),
+    );
+  }
+
+  buildProfilePhoto() {
+    return Container(
+      width: 60.0, 
+      height: 60.0, 
+      decoration: BoxDecoration(
+        color: kSecondaryColor, 
+        shape: BoxShape.circle
+      )
+    );
+  }
+
+  buildFullName() {
+    return Container(
+      margin: const EdgeInsets.only(top: 10.0),
       child: Column(
         mainAxisAlignment: MainAxisAlignment.center,
-        children: <Widget> [
-          Container(
-            width: 60.0, 
-            height: 60.0, 
-            decoration: BoxDecoration(
-              color: Color(0xFF45CAB9), 
-              shape: BoxShape.circle
-            )
-          ),
-          Container(
-            margin: const EdgeInsets.only(top: 10.0),
-            child: Column(
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: <Text> [
-                Text(
-                  fullName,
-                  style: GoogleFonts.muli(
-                    textStyle: TextStyle(
-                      color: Colors.white, 
-                      letterSpacing: .5, 
-                      fontSize: 30.0, 
-                      fontWeight: FontWeight.bold
-                    ), 
-                  )
-                ), 
-                Text(
-                  emailAddress, 
-                  style: GoogleFonts.muli(
-                    textStyle: TextStyle(
-                      color: Colors.white,  
-                      letterSpacing: .5, 
-                      height: 1.5, 
-                      fontSize: 14.0, 
-                    ), 
-                  )
-                )
-              ]
+        children: <Text> [
+          Text(
+            fullName,
+            style: GoogleFonts.muli(
+              textStyle: TextStyle(
+                color: Colors.white, 
+                letterSpacing: .5, 
+                fontSize: 20.0, 
+                fontWeight: FontWeight.bold
+              ), 
             )
           )
         ]
+      )
+    );
+  }
+
+  buildPopUpMenuButton(BuildContext context){
+    final IconData arrowDown = const IconData(0xf3d0,
+          fontFamily: CupertinoIcons.iconFont,
+          fontPackage: CupertinoIcons.iconFontPackage);
+
+    return ConstrainedBox(
+      constraints: BoxConstraints(maxWidth: 30.0, maxHeight: 30.0),
+      child: PopupMenuButton(
+        padding: EdgeInsets.all(0.0),
+        offset: Offset(MediaQuery.of(context).size.width * 0.375, 30),
```

Comment:
may want to make the `0.375` a constant/variable to make it more clear what this is

Reply:
Good idea!

Reply:
Whoops, just realized this is part of the popup menu stuff, so I'll just leave this as is for now! and I'll get rid of the whole thing in my next PR